addons/pkg/types/addons.go

```diff
@@ -63,3 +63,32 @@ type Image struct {
 	ImagePath string `yaml:"imagePath"`
 	Tag       string `yaml:"tag"`
 }
+
+// UpgradeDataValues contains the package nodeSelector and update strategy information
+type UpgradeDataValues struct {
```

Comment:
This is also used for install. I would call it TKGSDataValues.

Reply:
renamed